Makefile

```diff
@@ -1,5 +1,6 @@
 DIST := dist
 IMPORT := code.gitea.io/gitea
+export GO111MODULE=off
```

Comment:
Wouldn't this overwrite anything set by the user?

Reply:
Right. And it's intended.
Otherwise, some external tools would not work (those that do not support modules yet).